src/main/java/uk/gov/ons/ctp/integration/rhsvc/service/DataInCloud.java

```diff
@@ -0,0 +1,14 @@
+package uk.gov.ons.ctp.integration.rhsvc.service;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import uk.gov.ons.ctp.integration.rhsvc.domain.model.CaseContext;
+import uk.gov.ons.ctp.integration.rhsvc.domain.model.UACContext;
+
+import java.io.IOException;
+
+public interface DataInCloud {
```

Comment:
Ella had already created a RespondentDataService for this purpose - lets move this functionality into that